src/index.html

```diff
@@ -9,6 +9,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo"><img src="images/logo.png" alt="Logo: four letters meaning the name of the store MOYO"></a>
```

Comment:
```suggestion
      <a href="#" class="logo">
        <img src="images/logo.png" alt="Logo: four letters meaning the name of the store MOYO">
      </a>
```
Новый тег с новой строки